utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -146,7 +150,7 @@ int main(int argc, char **argv)
     int missingFuncFlag = 0;
     int i = 0, j = 0;
     int maxHeight = 0, maxWidth = 0;
-    int maxDstHeight = 0, maxDstWidth = 0;
+    //int maxDstHeight = 0, maxDstWidth = 0;
```

Comment:
Pls recheck if these variables are not used, and delete from Tensor_hip file too